release notes/v0.44.0.md

```diff
@@ -0,0 +1,224 @@
+- [#2901](https://github.com/grafana/k6/pull/2901). Added a warning when the number of time series exceeds 100000, which could lead to potential out-of-memory issues.
+- [#2997](https://github.com/grafana/k6/pull/2997). Added a new exit code (`109`), which signals about go panic.
+- [xk6-browser#788](https://github.com/grafana/xk6-browser/pull/788) Update the xk6-browser readme to highlight that it is now a module in k6.
+- [xk6-browser#803](https://github.com/grafana/xk6-browser/pull/803) Warns the user if the `browser.close` method is called more than once.
+- [xk6-browser#820](https://github.com/grafana/xk6-browser/pull/820) Add error handling to wildcard selectors, which cleans up the error output in the terminal.
+- [xk6-browser#843](https://github.com/grafana/xk6-browser/pull/843) Remove the build step from github actions. From this release onwards, no new standalone xk6-browser binaries will be built and available from the [releases](https://github.com/grafana/xk6-browser/releases) section. The latest version of the xk6-browser module will be available in the k6 binary which can be found [here](https://github.com/grafana/k6/releases).
+- [xk6-browser#848](https://github.com/grafana/xk6-browser/pull/848) Allow multiple k6 instances to connect to one browser to run concurrent tests. This update empowers high-concurrency k6 browser testing with multiple VUs and instances. Using the new `browserType.Connect` API, users can now connect to an existing browser instance and execute concurrent tests, which was not possible previously.
+
+## Bug fixes
+
+- [#2984](https://github.com/grafana/k6/pull/2984). Fixed wrongly assigned HTTP `POST` and `PUT` methods for the `tracing.instrumentHTTP`. Thanks, @marcin-maciej-seweryn!
+- [#2928](https://github.com/grafana/k6/pull/2928). Handled a new behavior of `filepath.Join` on windows go1.20, which could cause issues for the `k6 archive` command.
+- [#2915](https://github.com/grafana/k6/pull/2915). Fixed `check` that could return incorrect values for some cases with many preallocated VUs.
+- [#2953](https://github.com/grafana/k6/pull/2953). Fixed active VU reporting by arrival-rate executors.
+- [#2833](https://github.com/grafana/k6/pull/2833). Fixed detected memory leaks.
+- [#3006](https://github.com/grafana/k6/pull/3006). `xk6-websockets` updated to v0.2.0 which fixes a memory leak.
+- [xk6-browser#781](https://github.com/grafana/xk6-browser/pull/781) Fix mapping of `response` object's function from `jSON` to `json`.
+- [xk6-browser#779](https://github.com/grafana/xk6-browser/pull/779) Clear Zombie processes on panic.
+- [xk6-browser#834](https://github.com/grafana/xk6-browser/pull/834) Fix `page.close` so that it closes the current page and not the whole browser context.
+
+## Maintenance and internal improvements
+
+- [#2879](https://github.com/grafana/k6/pull/2879). Improved the per-VU buffer pool which should reduce memory usage and improve performance. Thanks, @davidpst!
```

Comment:
This likely should be higher and as part of the more highlighted changes. 

This likely means that a lot more people will now have pretty nice improvement in resource utilization.

Reply:
Fixed in 0c34d12.